### PR TITLE
Updates to support PSpell with TinyMCE 4.0.10

### PR DESCRIPTION
**Pull request by zymsys**
I also added a comment about Google's API being unavailable, and made a couple of small changes to the readme file.

### CODE REVIEW

**Comment by popthestack** [comment]
:+1:

**Comment by tcfunk** [comment]
:+1:

File: includes/PSpellSpellChecker.php [hunk 1]
```diff
@@ -48,8 +48,9 @@ public function getSuggestions($lang, $words) {
 
 		$outWords = array();
 		foreach ($words as $word) {
-			if (!pspell_check($plink, trim($word))) {
-				$outWords[] = utf8_encode($word);
+            $word = trim($word);
+            if (!pspell_check($plink, $word)) {
+                $outWords[utf8_encode($word)] = pspell_suggest($plink, $word);
```
Comment (mvanbaak) [inline includes/PSpellSpellChecker.php:53]: this will give broken output if pspell returns something else then utf-8 (which is the case in many languages)

**Comment by tinydylan** [comment]
This GitHub repository contains the server-side code for the TinyMCE Community SpellChecker Plugin. This plugin was deprecated with the release of TinyMCE 5.4. No more development will be done on this plugin, and we are closing all associated pull requests.

For more information, please see:

[TinyMCE Spellchecker: What are your Options](https://www.tiny.cloud/blog/tinymce-spellchecker)

[TinyMCE 5.4 Release Notes](https://www.tiny.cloud/docs/release-notes/release-notes54/#thefreetinymcespellcheckerplugin)

**Pull request closed without merging**